README.md

```diff
@@ -52,6 +52,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Eepos IT Services](http://www.eeposit.com/) | Old Baneshwor, Kathmandu | We are a full service mobile application, web, and custom software development company
 - [ESignature Pvt. Ltd.](http://esignature.com.np/) | Gyaneshwor, Kathmandu | Web-based solutions, web development, software solutions and IT service provider
 - [Expresiv Studios](http://expresivstudios.com/) | Dillibazar, Kathmandu | UI/UX Design for Mobile, Web & Beyond
+- [Exotrac](http://exotrac.com/) | Old Baneswor, Kathmandu | New York based leading provider of real time logistics management software for the automotive, manufacturing, retail, and supply chain logistics industries.
```

Comment:
Also can you not mention where the company is based. I am trying to avoid that. Also, remove the full stop at the end of the description